docs/phdata.rst

```diff
@@ -81,6 +81,8 @@ Optionally ``nanotimes_specs`` can also contain:
 -  **calibration_hist**: (array of integers) Histograms of
    uncorrelated counts used to correct the TCSPC non-linearities.
 
+(**this needs to contain some information on how these histograms have been acquired: duration of measurement, sample, optical setup configuration, etc)**
+
 Finally, if the data come from a simulation, ``/photon_data`` may contain:
```

Comment:
Do we really want to go this route, vs a separate file?

Would be acceptable to add a comment field (for each IRF or calibration hist) and leave to the user the level of book-keeping that he wants to achieve?